Source/WebCore/Modules/highlight/HighlightRegister.h

```diff
@@ -55,10 +55,12 @@ class HighlightRegister : public RefCounted<HighlightRegister> {
     
     WEBCORE_EXPORT void addAnnotationHighlightWithRange(Ref<StaticRange>&&);
     const HashMap<AtomString, Ref<Highlight>>& map() const { return m_map; }
+    const Vector<AtomString>& vector() const { return m_orderedHighlightNames; }
```

Comment:
This should have a more descriptive name than just 'vector', especially since it's an added bit of implementation detail different from he spec. I suggest orderedHighlightNames()